daniel-cabero/lesson_4/home_work/Mage.java

```diff
@@ -0,0 +1,25 @@
+package lesson_4.home_work;
+
+/**
+ * @author Daniel Cabero
+ * @since 09/12/2018.
+ */
+public class Mage extends Character implements Attack {
```

Comment:
these classes don´t have to implement IAttack interface because Charactarer should